.github/workflows/sync-testsite-branch.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync Main to Storybook Site Branch
```

Comment:
nit: name still refers to Storybook Site

Reply:
oops